workshop/content/100-setup/3-access-cluster.adoc

```diff
@@ -1,14 +1,14 @@
 :markup-in-source: verbatim,attributes,quotes
 
-== Create an Admin User for Cluster Access
+== Create a Local Admin User for Cluster Access
```

Comment:
Local? This is still in ROSA. I'm not sure I understand the motivation here.

Reply:
I suppose I mean as opposed to a user managed in an IDP ("outside" of ROSA). The Admin user here is managed via htpasswd which is "local" (ie internal) to ROSA. "Local" is probably too ambiguous ... would "locally managed" work better? Or "internal" work better? Open to whatever other options you have. Thanks!